apps/explorer/src/components/module/PkgModulesWrapper.tsx

```diff
@@ -117,15 +126,22 @@ function PkgModuleViewWrapper({ id, modules }: Props) {
         </div>,
         <div
             key="execute"
-            className="grow overflow-auto border-gray-45 pt-5 md:pl-7"
+            className="h-full grow overflow-auto border-gray-45 pt-5 md:pl-7"
         >
             <TabGroup size="md">
                 <TabList>
                     <Tab>Execute</Tab>
                 </TabList>
                 <TabPanels>
                     <TabPanel>
-                        <div className="h-verticalListLong overflow-auto">
+                        <div
+                            className={clsx(
+                                'overflow-auto',
+                                (splitPanelOrientation === 'horizontal' ||
+                                    !isMediumOrAbove) &&
+                                    'h-verticalListLong'
```

Comment:
fixed height only needs to appear for `horizontal` or mobile view. Otherwise it will be larger than the draggable container.